.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+version: 2.1
```

Comment:
This code shouldn't be implemented in develop branch yet

Reply:
Colocamos esta parte do código apenas para a PR ser aceita pela pypeline, mas vamos remover então